src/frontends/onnx/frontend/src/editor.cpp

```diff
@@ -175,86 +175,6 @@ void modify_initializer(TensorProto& initializer,
         tensor_type->set_elem_type(initializer.data_type());
     }
 }
-
-bool is_topologically_sorted(const GraphProto& graph) {
```

Comment:
ONNX FE should be topologically sorted according to the standard, but we try to be more user-friendly. Why this change is removed?